src/main/java/org/jahia/commons/encryption/ConfigurationUtils.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * ==========================================================================================
+ * =                   JAHIA'S DUAL LICENSING - IMPORTANT INFORMATION                       =
+ * ==========================================================================================
+ *
+ *                                 http://www.jahia.com
+ *
+ *     Copyright (C) 2002-2019 Jahia Solutions Group SA. All rights reserved.
```

Comment:
Never know if that's important or not 🤷‍♂️ :
```suggestion
 *     Copyright (C) 2002-2025 Jahia Solutions Group SA. All rights reserved.
```